docs/enhancements/template.md

```diff
@@ -26,6 +26,16 @@ To get started with this template:
 The `Metadata` section above is intended to support the creation of tooling
 around the enhancement process.
 
+Make sure that the enhancement covers the following scenarios:
```

Comment:
Should we add those as subsections to "scenarios"?
I'm split on that one though, maybe it's too cumbersome